app/views/errors/internal_server_error.html.haml

```diff
@@ -0,0 +1,33 @@
+%html
+  %head
+    %title Page not found!
+    %meta{:content => "width=device-width,initial-scale=1", :name => "viewport"}/
+    :css
```

Comment:
There shouldn't be any inline CSS. Move this to `error.sass` file and include it in the stylesheet. If you want to target the body with CSS, add controller and action name as a class in the body tag in the layouts. Something like `body{class: "c-" + params[:controller] + " a-" + params[:action]}`